themes/copper-hugo/layouts/contact/list.html

```diff
@@ -14,19 +14,31 @@
         <form method="POST" class="rounded shadow p-30" action="{{site.Params.contact_form_action}}">
           <div class="form-group mb-30">
             <label class="text-black-200 h5 font-weight-600 mb-2" for="fname">{{ i18n "name"}}</label>
-            <input class="form-control shadow-none" type="text" placeholder="Jack Barker" id="fname" required>
+            <input class="form-control shadow-none" type="text" placeholder="Jack Barker" id="fname" name="fname" required>
           </div>
           <div class="form-group mb-30">
             <label class="text-black-200 h5 font-weight-600 mb-2" for="email">{{i18n "mail"}}</label>
-            <input class="form-control shadow-none" type="email" placeholder="barker@Copper.io" id="email" required>
+            <input class="form-control shadow-none" type="email" placeholder="barker@Copper.io" id="email" name="email" required>
           </div>
           <div class="form-group mb-30">
             <label class="text-black-200 h5 font-weight-600 mb-2" for="website">{{i18n "website"}}</label>
-            <input class="form-control shadow-none" type="url" placeholder="Website">
+            <input class="form-control shadow-none" type="url" id="website" name="website" placeholder="Website">
+          </div>
+          <div class="form-group mb-30">
+            <label for="reason" class="text-black-200 h5 font-weight-600 mb-2">{{ i18n "contact_reason" }}</label>
```

Comment:
This seems to cut off on my end, Chrome on Pop!_OS
![image](https://user-images.githubusercontent.com/12040685/131418080-bed88fa8-3c9c-4853-8cce-2f5dc30517d3.png)

Can we make it just a bit taller?

Reply:
Yes, but let's push this fix in if you don't mind as right now we are dropping contacts.